app/build.gradle

```diff
@@ -333,6 +336,10 @@ android {
             manifest.srcFile 'CommcareAndroidManifest.xml'
         }
 
+        cccStaging {
+            manifest.srcFile 'CommcareAndroidManifest.xml'
```

Comment:
is this required ?

Reply:
Hmm... good question! I assumed so since it's specified instead of the default for both commcare and lts, but looking into CommcareAndroidManifest.xml it seems like maybe it's just for adding some extra shortcuts?

Reply:
I can try removing it and see if things work, although I'm not sure what error to look for other than just making sure the app installs and launches successfully.